install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -707,8 +707,18 @@ enableCiliumEndpointSlice: false
 ciliumEndpointSlice:
   # -- Enable Cilium EndpointSlice feature.
   enabled: false
-  # -- Configure a custom dynamic rate limit for the CiliumEndpointSlice controller.
-  dynamicRateLimitConfig: ""
+  # -- List of rate limit options to be used for the CiliumEndpointSlice controller
+  dynamicRateLimitConfig: []
+  # A custom dynamic rate limit configuration can be provided for the
+  # CiliumEndpointSlice controller. Each object in the list must have a
+  # node count, limit, and burst value.
+  # dynamicRateLimitConfig:
+  # # -- Count of nodes at which to apply this rate limit
+  # - nodes: 0
+  # # -- The sustained request rate in requests per second
+  #   limit: 10
+  # # -- The burst request rate in requests per second
+  #   burst: 20
```

Comment:
Looking at the code, it seems that the maximum allowed values are respectively 50 and 100, which may be worth documenting.

Reply:
Added in the comments. I was hoping helm-docs supported docstrings for fields of an object within a list, but I think that may beyond what the tool can handle. Added details for the fields within the top-level comment and moved the default config to the helm values..